### ISSUE TITLE
[Feature Req] tp out of faction controlled land on kick

### ISSUE THREAD

**Issue by chausser** [MassiveCraft/Factions#36]
It would be nice to have plugin teleport a player who has been kicked from a faction(only if they are in the game of course) to the first available wilderness block. We have some users inviting people to factions and once they are in the land kicking them and killing them. Just a thought. -Chausser
